types/Projects.ts

```diff
@@ -127,3 +128,10 @@ export type Component<T = GenericComponentConfig> = {
   runnable: boolean;
   path: string;
 };
+
+export type ProjectDevTargetAccountPromptResponse = {
```

Comment:
Should this live in the prompt file instead?

Reply:
I couldn't decide between the two, but if you vote for prompts lets go with that